src/EnergyPlus/ThermalComfort.cc

```diff
@@ -2085,7 +2085,7 @@ namespace ThermalComfort {
         return CalcAngleFactorMRT;
     }
 
-    Real64 CalcSurfaceWeightedMRT(EnergyPlusData &state, int const ZoneNum, int const SurfNum)
+    Real64 CalcSurfaceWeightedMRT(EnergyPlusData &state, int const ZoneNum, int const SurfNum, bool AverageWithSurface = true)
```

Comment:
You should modify the purpose description in the comment below to reflect the change here.